index.js

```diff
@@ -55,23 +55,27 @@ function callCumulusMessageAdapter(command, input) {
  * If a Cumulus Remote Message is passed, fetch it and return a full Cumulus Message
  *
  * @param {Object} cumulusMessage - either a full Cumulus Message or a Cumulus Remote Message
+ * @param {String} schemaLocation - location of schema file, can be null
```

Comment:
The schema location should be an Object that includes Strings for the input, output, and config schemas.